lib/util.ts

```diff
@@ -1,16 +1,21 @@
-'use strict';
-const fs = require('fs');
-const thrift = require('thrift');
-const parquet_thrift = require('../gen-nodejs/parquet_types')
-
+import thrift from "thrift";
+import fs from 'fs'
+import parquet_thrift from '../gen-nodejs/parquet_types'
 
 /** We need to use a patched version of TFramedTransport where
   * readString returns the original buffer instead of a string if the 
   * buffer can not be safely encoded as utf8 (see http://bit.ly/2GXeZEF)
   */
 
 class fixedTFramedTransport extends thrift.TFramedTransport {
-  readString(len) {
+  inBuf: Buffer
+  readPos: number
+  constructor(inBuf: Buffer) {
+    super(inBuf)
+    this.inBuf = inBuf
+    this.readPos = 0
+  }
+  readString(len: number) {
```

Comment:
The super class has a readString function but this does not override the superclass one. It is instead conflicting with the typing of the superclass readString and I'm not sure why.